common-util/useFetchApi.js

```diff
@@ -13,12 +13,23 @@ const useFetch = (subUrl, params) => {
   return { data, isLoading };
 };
 
-export const useFetchVideos = (limit = 1000) => {
-  const params = {
-    sort: ['date:desc'],
-    populate: '*',
-    'pagination[limit]': limit,
-  };
+export const useFetchVideos = (limit = 1000, isPodcast = false) => {
```

Comment:
would be nice to make params an object, so that when in use it's easier to read this `useFetchVideos({ limit: LIMIT, isPodcats: true })` rather than this `useFetchVideos(LIMIT, true)`